specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2018-02-01-preview/containerInstance.json

```diff
@@ -314,6 +321,39 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerInstance/locations/{location}/operations/{operationId}": {
+      "get": {
+        "operationId": "AsyncContainerGroupOperation_Get",
+        "x-ms-examples": {
+          "ContainerUsage": {
+            "$ref": "./examples/ContainerGroupUsage.json"
+          }
+        },
+        "description": "Get the usage for a subscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
```

Comment:
I really don't understand what you'are trying to do here :(
Even the example "ContainerGroupUsage" doesn't have this information

Reply:
The example was incorrect. The latest commit fixed this issue.